.github/workflows/test.yml

```diff
@@ -12,6 +12,8 @@ jobs:
     strategy:
```

Comment:
You should change also the `name: test_py37` line to some general name that matches also the badge i think.